### PR TITLE
Update spanish translations

### PR DESCRIPTION
**Pull request by beausmith**
I pushed an NPM patch release for for 1.3.2 and deprecated 1.3.1. In hindsight, I probably didn't need to deprecate 1.3.1.
```
beau@vx ~/Development/ballot-encoder update-spanish-translations
❯ npm version patch
v1.3.2
beau@vx ~/Development/ballot-encoder update-spanish-translations
❯ yarn npm publish --access=public
... snip ...
beau@vx ~/Development/ballot-encoder update-spanish-translations
❯ npm deprecate -f '@votingworks/ballot-encoder@1.3.1' "Poor translations"
```

### CODE REVIEW

File: src/data/electionSampleLongContent.json
Comment (beausmith) [inline src/data/electionSampleLongContent.json:1028]: I'll make this a smart apostrophe next version

**Review by eventualbuddha — APPROVED** [review]
LGTM

**Comment by eventualbuddha** [comment]
In general, I'd hold off publishing a version until you merge the PR. Otherwise you could end up making a change that requires you to bump the version again.

**Comment by beausmith** [comment]
Yes. Makes sense. Got a little excited with the new knowledge you dropped.

**Pull request merged**
